src/main/java/org/broadinstitute/hellbender/cmdline/CommandLineProgram.java

```diff
@@ -280,6 +278,24 @@ public final void setDefaultHeaders(final List<Header> headers) {
     }
 
     /**
+     * Print a curated set of important HTSJDK default settings to the logger
+     */
+    protected void printHTSJDKDefaults() {
```

Comment:
Thanks for making this protected and package-protected/final... This will allow to print specific defaults for other toolkits dependencies and/or reduce verbosity.

Just a suggestion: maybe it is better to rename to specify that this could also log other stuff if implementors require it...